imports/ui/pages/track/tracktime.js

```diff
@@ -424,6 +426,17 @@ Template.tracktime.helpers({
   holidayToday: () => (isHoliday(Template.instance().date.get())
     ? isHoliday(Template.instance().date.get())[0].name : false),
   replaceSpecialChars: (string) => string.replace(/[^A-Z0-9]/ig, '_'),
+  logForOtherUsers: () => {
+    if (getGlobalSetting('enableLogForOtherUsers') && Template?.instance()?.projectId?.get()) {
+      const targetProject = Projects.findOne({ _id: Template.instance().projectId.get() })
+      if (targetProject
```

Comment:
I think it would be great if we only show the user search if there are other users assigned to the project, maybe we can somehow retrieve the userslist and store it in a reactivevar that we can check here as well?

Reply:
Fixed.